static/src/javascripts/projects/common/modules/article/twitter.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import fastdom from 'fastdom';
+import config from 'lib/config';
+import { getBreakpoint, getViewport } from 'lib/detect';
+import mediator from 'lib/mediator';
+import debounce from 'lodash/functions/debounce';
+
+declare var twttr: Object;
```

Comment:
was the only way I could push my branch 🤔

Reply:
We have a [global type module](https://github.com/guardian/frontend/blob/master/static/src/javascripts/__flow__/types/global.js), perhaps this would be a better place to define this?

Reply:
moved it there but now am getting lots of errors. anything I should do to make it work?

Reply:
scratch that, it's working now 😄